app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -255,6 +256,18 @@ def networks
     end
   end
 
+  def placement_groups
+    collector.placement_groups.placement_groups.each do |sgrp|
+      persister.placement_groups.build(
+        :type              => "ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::PlacementGroup",
```

Comment:
The type should be automatically detected and this shouldn't be necessary (if it is that means something else is wrong)